test/unit/utils/TagUtils.ts

```diff
@@ -0,0 +1,141 @@
+import test from "ava";
+import TagUtils from '../../../src/utils/TagUtils';
+import _ from "lodash";
+import sinon, { SinonSandbox } from "sinon";
+import { TestEnvironment } from '../../support/sdk/TestEnvironment';
+import { TagCategory, TagsObjectForApi, TagsObjectWithBoolean, Categories } from '../../../src/models/Tags';
+import { deepCopy } from '../../../src/utils';
+
+const sinonSandbox: SinonSandbox = sinon.sandbox.create();
+
+test.beforeEach(async () => {
+    await TestEnvironment.initialize({ stubSetTimeout: true });
+    TestEnvironment.mockInternalOneSignal();
+});
+
+test.afterEach(() => {
+    sinonSandbox.restore();
+});
+
+test('check conversion from tag boolean values to number strings', t => {
+    const converted = TagUtils.convertTagsBooleansToApi({ tag1: false, tag2: true });
+    t.true(_.isEqual(converted, { tag1: "0", tag2: "1" }));
```

Comment:
This line could be simplified with `t.is()` instead of using ` t.true(_.isEqual())`
```typescript
t.is(converted, { tag1: "0", tag2: "1" });
```

Reply:
Done.